applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication_mpi.py

```diff
@@ -0,0 +1,50 @@
+# import Kratos
+import KratosMultiphysics
+
+# Import Kratos "wrapper" for unittests
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+try:
+    import KratosMultiphysics.mpi as KratosMPI
```

Comment:
I guess it would be a good idea to also import metris and trilinos here (I should also do this in StructuralMechanics)

The solvers need it anyway and the error will be more clear if it fails here

Reply:
done :)